components/bitly/common/common.mjs

```diff
@@ -0,0 +1,54 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  methods: {
+    async createBitlink($, payload, oauth_access_token) {
```

Comment:
This method should be included in the app file, where you have access to `this.$auth`. That way you don't need to pass the OAuth access token from the caller. See e.g. https://github.com/PipedreamHQ/pipedream/blob/026cdf37f6832c6dfa7467c37898c5cd377a1cf6/components/gitlab/gitlab.app.mjs#L151